pom.xml

```diff
@@ -131,6 +134,11 @@
             <artifactId>commons-compress</artifactId>
             <version>1.21</version>
         </dependency>
+        <dependency>
+            <groupId>org.xerial.snappy</groupId>
+            <artifactId>snappy-java</artifactId>
+            <version>${snappy.version}</version>
```

Comment:
this should be in dependencyManagement because we don't use it directly

Reply:
Moved.